README.md

```diff
@@ -33,28 +54,212 @@ import (
 	"fmt"
 	"log"
 	"net"
-
 	"github.com/ipinfo/go/v2/ipinfo"
 )
 
 func main() {
-	// Get access token by signing up a free account at https://ipinfo.io/signup
-	client := ipinfo.NewClient(nil, nil, "MY_TOKEN")
-	info, err := client.GetIPInfo(net.ParseIP("8.8.8.8"))
+	const token = "YOUR_TOKEN"
+	
+	// params: httpClient, cache, token. You can set the first two params as `nil`
```

Comment:
```suggestion
	// params: httpClient, cache, token. `http.DefaultClient` and no cache will be used in case of `nil`.
```

Reply:
We should specify what will be selected in case of nil as this statement might confuse some users.